src/eventHandlers/orderEventHandler.ts

```diff
@@ -0,0 +1,47 @@
+import logger from '../logger.js'
+import { publishMessage } from '../pubsub.js'
+import { ENTUR_POS_NATIVE, ENTUR_POS_WEB } from '../config.js'
+import eventsWhitelist from '../eventsWhitelist.js'
+import { removeEventNameLevelFromEvent } from './utils.js'
+
+const isForSelfService = (eventContents: any): boolean => {
+    const pos = eventContents.meta?.pos
```

Comment:
Hvis dette er en fast plassering for pos så kan vi kanskje trekke det ut i utils (siden tilnærmet samme kode finnes i paymentEventHandler)?